janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/resources/hbase-read.properties

```diff
@@ -0,0 +1,29 @@
+gremlin.graph=org.apache.tinkerpop.gremlin.hadoop.structure.HadoopGraph
+gremlin.hadoop.graphInputFormat=org.janusgraph.hadoop.formats.hbase.HBaseInputFormat
+gremlin.hadoop.graphOutputFormat=org.apache.hadoop.mapreduce.lib.output.NullOutputFormat
+#gremlin.hadoop.graphOutputFormat=org.apache.hadoop.mapreduce.lib.output.SequenceFileOutputFormat
```

Comment:
Do we need this commented-out line? Is there a use case to prefer this over the `NullOutputFormat` on the previous line? Does this help with debugging test failures somehow?